spec/yaml/swiftnav/sbp/flash.yaml

```diff
@@ -15,7 +15,7 @@ description: >
   Swift Navigation devices: the STM32 flash and the M25Pxx FPGA
   configuration flash from Piksi 2.3.1.  This module does not apply
   to Piksi Multi.
-stable: False
+stable: True
```

Comment:
Same here, this is for Piksi v2, so it has little value to current customers so I don't see any benefit in marking it as stable, we should probably add a "deprecated" section instead of promoting this to stable.